pkg/scanner/local/scan.go

```diff
@@ -389,14 +389,16 @@ func (s Scanner) scanLicenses(target types.ScanTarget, options types.ScanOptions
 	for _, app := range target.Applications {
 		var langLicenses []types.DetectedLicense
 		for _, lib := range app.Libraries {
-			for _, license := range lib.Licenses {
-				category, severity := scanner.Scan(license)
+			for _, license := range lib.LicensesV2 {
+				category, severity := scanner.Scan(license.Name)
 				langLicenses = append(langLicenses, types.DetectedLicense{
-					Severity:   severity,
-					Category:   category,
-					PkgName:    lib.Name,
-					Name:       license,
-					Confidence: 1.0,
+					Severity:    severity,
```

Comment:
We need the pkgVersion to link the license to the pkg

Reply:
sure